runtime/common/src/auctions.rs

```diff
@@ -1087,23 +1038,28 @@ mod tests {
 	#[test]
 	fn can_win_auction() {
 		new_test_ext().execute_with(|| {
+			MockLeaser::mock_lease_period_index(|index| {
+				assert_eq!(index, 1);
+				Some((0, 0))
+			});
+			MockLeaser::mock_deposit_held(|para, bidder| {
+				assert_eq!(bidder, 1);
+				assert_eq!(para, 0);
+				0
+			});
+			MockLeaser::mock_already_leased(|para, first_slot, last_slot| {
+				assert_eq!(para, 0);
+				assert_eq!(first_slot, 1);
+				assert_eq!(last_slot, 4);
+				false
+			});
```

Comment:
If these mocks represent the "common" behavior, they can be placed under `new_test_ext()` and shared them across all test cases. Later each test case can overwrite the mock they need for its specific test purpose.

Also, it's not mandatory to check any input parameter, and in most cases is enough just return the desired value:

```rust
MockLeaser::mock_lease_period_index(|_| Some((0, 0)));
MockLeaser::mock_deposit_held(|_, _| 0);
MockLeaser::mock_already_leased(|_, _, _| false);
```